### PR TITLE
Updated test results and fixes for PTS 7.0

### PR DESCRIPTION
**Pull request by michal-narajowski**

### CODE REVIEW

File: apps/bletiny/src/gatt_svr.c [hunk 1]
```diff
@@ -134,7 +134,7 @@ static const struct ble_gatt_svc_def gatt_svr_svcs[] = {
             }, {
                 .uuid128 = PTS_UUID(PTS_CHR_WRITE_NO_RSP),
                 .access_cb = gatt_svr_access_test,
-                .flags = BLE_GATT_CHR_F_WRITE_NO_RSP,
+                .flags = BLE_GATT_CHR_F_READ | BLE_GATT_CHR_F_WRITE_NO_RSP,
```
Comment (michal-narajowski) [inline apps/bletiny/src/gatt_svr.c]: Thanks, Łukasz. Updated commit msg and rebased.

**Comment by ccollins476ad** [comment]
Hi Michał,

Everything looks great.  Unfortunately a few of the host unit tests are failing.  I see what the problem is: you fixed a bug in ble_hs_pvcy_set_addr_timeout(), and the tests are expecting the bug to still be present.  Specifically, the unit tests are not simulating an incoming command complete event to acknowledge the  LE Set Resolvable Private Address Timeout Command.

Fixing the tests might be tedious and annoying.  The command complete should only be simulated when the test uses privacy.

Thanks,
Chris

**Comment by michal-narajowski** [comment]
Hi Chris,

Fixed the tests.

Thanks,
Michał

**Comment by ccollins476ad** [comment]
Looks good.  Thanks, Michał.

**Pull request merged**